ctwrap/simulation.py

```diff
@@ -59,15 +69,21 @@ def __init__(self, module: str, output: Dict[str, Any] = None):
     def from_module(cls,
                     module: str,
                     output: Optional[Dict[str, Any]] = None) -> \
-            Union['simulation', Dict[str, Any], str]:
+            Union['Simulation', Dict[str, Any], str]:
         """
         Alternative constructor for `Simulation` object.
-        The `from_module` instantiation call renames the class
-        based on the module name.
+
+        The :meth:`from_module` method is intended as the main route for the creation of
+        `Simulation` objects. For example:
+
+        .. code-block:: Python
+
+           # creates a simulation object with module `minimal`
+           sim = ctwrap.Simulation.from_module(cw.modules.minimal)
 
         Arguments:
            module (module): handle name or handle to module running the simulation
-           output (dict): generated by `SimulationHandler`
+           output (dict):  Output information. Generated by `SimulationHandler`
```

Comment:
The `output` probably needs some additional discussion (in #28?) as only HDF is left.